packages/kernel/handlers/aptosHandler.ts

```diff
@@ -0,0 +1,129 @@
+import { getAptosConnection } from '@walless/engine/crawlers/aptos';
+import { ResponseCode } from '@walless/messaging';
+import {
+	APTOS_COIN,
+	AptosAccount,
+	CoinClient,
+	FungibleAssetClient,
+	HexString,
+	TokenClient,
+} from 'aptos';
+import { HandleMethod } from '../utils/types';
+
+export const handleTransferCoin: HandleMethod = async ({
+	privateKey,
+	payload,
+	responseMethod,
+}) => {
```

Comment:
`HandleMethod` only works on `apps/web`, therefore these methods should handle only on-chain action and return result like signature or error, could ask @tanlethanh or reference other chains for more information